ui/src/dashboards/components/EditVEO.tsx

```diff
@@ -23,30 +23,30 @@ import {AppState, RemoteDataState, QueryView, TimeMachineID} from 'src/types'
 interface DispatchProps {
   onSetName: typeof setName
   onSaveView: typeof saveVEOView
-  getViewForTimeMachine: typeof getViewForTimeMachine
+  setQueryResultsForCell: typeof setQueryResultsForCell
 }
 
 interface StateProps {
-  view: QueryView | null
   activeTimeMachineID: TimeMachineID
+  view: QueryView | null
 }
 
 type Props = DispatchProps & StateProps & WithRouterProps
 
 const EditViewVEO: FunctionComponent<Props> = ({
-  getViewForTimeMachine,
   activeTimeMachineID,
   onSaveView,
   onSetName,
   params: {orgID, cellID, dashboardID},
   router,
+  setQueryResultsForCell,
   view,
 }) => {
   useEffect(() => {
     // TODO split this up into "loadView" "setActiveTimeMachine"
     // and something to tell the component to pull from the context
     // of the dashboardID
-    getViewForTimeMachine(dashboardID, cellID, 'veo')
+    setQueryResultsForCell(dashboardID, cellID, 'veo')
```

Comment:
ok this is a nit, but this function name is a lot less descriptive to me. can we switch it to something that has a reference to view? maybe `getViewAndResultsForVEO`?